eng/Versions.props

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <!-- Experiment package versions properties -->
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <!-- File version numbers -->
+    <MajorVersion>1</MajorVersion>
+    <MinorVersion>0</MinorVersion>
+    <PatchVersion>0</PatchVersion>
+    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
```

Comment:
We probably will want to avoid version clashes so my suggestion here would be to do the same we do in not standalone experiments and set the label as the experiment name instead.

Reply:
Granted its unlikely to have clashes with package names, but one other reason is to easily tell from which branch was a package produced when just looking at the feed.

Reply:
I left it this way since experiments will be in their own branch and the name of the package should tell which experiment it is... in the other kind of experiments since we produce a shared framework it makes sense, but in my opinion here I would just leave it to the experiment owner to use the label they want.